src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
                 return new Session($content);
             },
+            'users/signup' => function (array $content) {
```

Comment:
Yes we have this endpoint. https://github.com/netmedialtd/app_rebillyx/blob/development/api/config/params.php#L381
- here all endpoints for this PR